### DISCUSSION TITLE
After Filament login in, js and css file reports error 500.

### DISCUSSION THREAD

**Discussion by dogeow** [filamentphp/filament#6255] [Help]
Hi.

When the defaults guard is set to `web`, it is normal, but `$user = auth()->user();` must be written as `$user = auth()->guard('sanctum') ->user();`, it is cumbersome.

The error occurs when the defaults guard of config/auth.php is set to `sanctum`:

<img width="1421" alt="image" src="https://user-images.githubusercontent.com/7204604/232999668-9bebc87c-f92c-4bbe-b34b-6422e281515c.png">

How can i  use sanctum by default.

**Comment by danharrin** [comment]
Why would you use Sanctum to authenticate Filament? Sanctum is for API tokens and Filament doesn't use an API.

**Reply by dogeow** [reply]
I use sanctum to provide the api for the vue front end, which is the route in config/api.php. Like user login, logout etc.

**Reply by danharrin** [reply]
But Filament uses session authentication. We don't have an API so it doesnt make sense for us to use API tokens

**Comment by carestad** [comment]
Same here. Filament reports 500 errors for CSS and JS assets, even though I have specified in the `filament.php` config file that _Filament_ needs to use the `web` guard. I do not use Sanctum to auth towards Filament, but it is the default for everything else in the application.
